react-accordion-component/my-app/src/App.jsx

```diff
@@ -0,0 +1,31 @@
+// import { useState } from 'react'
+import './App.css';
+import Accordion from './Accordion';
+
+const arrayTopic = [
```

Comment:
FYI: I would name this variable `arrayTopics` with an s. Since we have more than one topic, this is more clear!